pkgs/tools/package-management/microdnf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, gettext, libdnf, pkg-config, glib, libpeas, libsmartcols }:
+
+stdenv.mkDerivation rec {
+  pname = "microdnf";
+  version = "3.6.0";
+
+  src = fetchFromGitHub {
+    owner = "rpm-software-management";
+    repo = pname;
+    rev = version;
+    sha256 = "0a7lc3qsnblvznzsz3544l3n84184xi85zf7c3m3jhnmpmxsg39h";
+  };
+
+  nativeBuildInputs = [ pkg-config cmake gettext libdnf ];
+  buildInputs = [ libdnf glib libpeas libsmartcols ];
+
+  # Remove manpage generation which dies because of meson issues addressed in postInstall.
```

Comment:
This comment is disconnected now since there is not `postInstall` and `meson` is not used at all. Maybe see if replacing `cmake` for `meson` and `ninja` actually fixes the issue.

Reply:
It did indeed, thanks for catching that